package.json

```diff
@@ -188,18 +188,15 @@
                 },
                 "bazel.lsp.command": {
                     "type": "string",
-                    "default": "starlark",
+                    "default": "",
```

Comment:
One thing I was wondering: Should we maybe also remove the `bazel.lsp.enabled` setting? And instead just enable the LSP as soon as `bazel.lsp.command` is set to a non-empty string.

With this pull request, I now need to set only two instead of three settings (I no longer need to set the `bazel.lsp.args`). If we also remove `bazel.lsp.enabled`, we could cut down the setup effort to a single setting.

Reply:
Yeah I think this makes sense.